pages/common/plesk.md

```diff
@@ -0,0 +1,28 @@
+# plesk
+
+> Plesk hosting control panel CLI interface.
+> More information: <https://docs.plesk.com/>.
+
+- Generate auto login link for admin user:
```

Comment:
```suggestion
- Generate a auto login link for the admin user:
```

Reply:
`an` is the appropriate article before a word that starts with a vowel, @marchersimon, so `an auto` would be correct here.

Reply:
Whoops, edited.